onnxruntime/core/optimizer/attention_fusion_helper.h

```diff
@@ -593,6 +610,52 @@ bool MatchInputMaskSubgraph(const Graph& graph, const Node& qkv_matmul, Attentio
   return true;
 }
 
+bool MatchInputMaskSubgraph(const Graph& graph, const Node& qkv_matmul, AttentionMaskNodesDistilBert& result, const logging::Logger& logger) {
+  DEBUG_LOG("Start MatchInputMaskSubgraphDistilBert");
+
+  std::vector<graph_utils::EdgeEndToMatch> mask_path{
+      {0, 0, "Softmax", {1, 11, 13}, kOnnxDomain},
+      {0, 0, "Where", {9}, kOnnxDomain},
+      {0, 0, "Expand", {8}, kOnnxDomain},
+      {0, 0, "Reshape", {5}, kOnnxDomain},
+      {0, 0, "Equal", {1, 11}, kOnnxDomain}};
```

Comment:
According to https://github.com/onnx/onnx/blob/master/docs/Operators.md, Equal has 13, 11, 7, 1.